hubspot.com.hubspot-cname-txt.json

```diff
@@ -19,7 +19,7 @@
     {
       "groupId": "cname",
       "type": "CNAME",
-      "host": "@",
+      "host": "%cNameHostValue%",
```

Comment:
So in case of subdomains you'd apply this template on APEX (without host param) with specific subdomain in `%cNameHostValue%` ?
This will work but you need to be aware of the limitation of 1 instance in the zone at once.
If you want to overcome this `multiInstance` feature would have to be enabled.

Also, DNS providers would most likely not accept an unprotected template with a variable in both host and pointsTo parameters of the CNAME.
Please read https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#security-considerations for the security threat consierations around it and related mitigation methods in the protocol.

Reply:
So, as I understand we can't set `"host": "@",` and we can't use the second variable.

The only option is to digitally sign requests

Reply:
Thanks for the tips.
>multiInstance
 I double-check with our PM. We actually only need to support subdomain. We will add txtHostValue variable
 
 >unprotected template 
 We can definitely add the signature to the request  
 
 
  

I do not see a reason to